internal/ingress/metric/collector/collector.go

```diff
@@ -272,6 +269,28 @@ func (sc *UDPCollector) handleMessage(msg []byte) {
 }
 
 // Run adds a message handler to a UDP listener
-func (sc *UDPCollector) Run() {
-	handleMessages(sc.listener, sc.handleMessage)
+func (sc *SocketCollector) Run() {
+	for {
+		conn, err := sc.listener.Accept()
+		if err != nil {
+			continue
+		}
+
+		go handleMessages(conn, sc.handleMessage)
+	}
+}
+
+const packetSize = 1024 * 65
+
+// handleMessages process packets received in an UDP connection
```

Comment:
Let's also update this comment to omit UDP.

Here's some other places as well: 
* https://github.com/kubernetes/ingress-nginx/pull/2652/files#diff-51900d33d0f20d5ddd66a3bd1b1727b9R271
* https://github.com/kubernetes/ingress-nginx/pull/2652/files#diff-51f1a91ea84da03f4bb49a5123753f54R138

Reply:
done